build.ps1

```diff
@@ -228,6 +228,11 @@ if ($All) {
     $MSBuildArguments += '/p:BuildAllProjects=true'
 }
 elseif ($Projects) {
+    if (![System.IO.Path]::IsPathRooted($Projects))
+    {
+        $CurrentDirectory = Get-Location;
+        $Projects = Join-Path $CurrentDirectory $Projects
```

Comment:
```suggestion
        $Projects = Join-Path (Get-Location) $Projects
```
you can avoid having a variable but just moving this inline. PowerShell is fun :)